scripts/auto_update_trajectories.py

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
let's wrap the operation of the script (esp part like modification of `dj.config['safemode']`) into `if __name__ == "__main__":` block.